### PR TITLE
Profile pic additions

### PR DESCRIPTION
**Pull request by bbryanb333**
added profile pic fetches for recent listings and profile view

### CODE REVIEW

**Comment by americobarros** [comment]
honestly not sure how to QA any of this, I mean it looks fine and if it wasn't coming up on the last few days maybe I would say move some more stuff to functions but whatever overall looks good

File: SH4RE/SH4RE/Home/HomeView.swift
Comment (americobarros) [inline SH4RE/SH4RE/Home/HomeView.swift:1]: The preview breaks when trying to preview home view.

File: SH4RE/SH4RE/Home/HomeView.swift [hunk 1]
```diff
@@ -125,6 +127,15 @@ struct HomeView: View {
                 fetchMainImage(listing: listing2, completion: { image2 in
                     recentListing2Image = image2
                 })
+                
+                chatLogViewModel1 = ChatLogViewModel(chatUser: ChatUser(id:listing1.uid, uid: listing1.uid, name: listing1.ownerName))
+                getProfilePic(uid: listing1.uid, completion: { profilePic in
+                    chatLogViewModel1?.profilePic = profilePic
+                })
+                chatLogViewModel2 = ChatLogViewModel(chatUser: ChatUser(id:listing2.uid, uid: listing2.uid, name: listing2.ownerName))
```
Comment (americobarros) [inline SH4RE/SH4RE/Home/HomeView.swift:153]: why do you have to get the profile pics on home view? isn't this something that would happen in viewlistingview or messages or profile view?

**Review by americobarros — CHANGES_REQUESTED** [review]
a questions then just a little bug

File: SH4RE/SH4RE/Home/HomeView.swift (hunk 1 above)
Comment (bbryanb333) [reply SH4RE/SH4RE/Home/HomeView.swift:153]: The way messaging is designed we need to instantiate a chatUser at the time we query the listing. So these are just creating the chatUser for the 2 recent listings

File: SH4RE/SH4RE/Home/HomeView.swift
Comment (bbryanb333) [reply SH4RE/SH4RE/Home/HomeView.swift:1]: I will look at this

Comment (americobarros) [reply SH4RE/SH4RE/Home/HomeView.swift:1]: seems to be happening on main too, might have something to do with recent posts as well or something that was inserted into the database

Comment (americobarros) [reply SH4RE/SH4RE/Home/HomeView.swift:1]: Screenshot 2023-04-04 at 12 03 10 PM

**Pull request merged**